default/scripting/ship_parts/Armour/AR_REINFORCED_HULL.focs.txt

```diff
@@ -0,0 +1,27 @@
+Part
+    name = "AR_REINFORCED_HULL"
```

Comment:
This is similar to the regeneration of living hulls, without the out-of-combat restriction...

Reply:
also similar to basic damage control and fleet field repair, except it is different in that it is coupled to a ship module that needs to be mounted in the core slot.

Reply:
How many different ways to repair ships are needed? What's notable (not just different) about this one?

Reply:
it gives more design options for capital ships with available core slots, since there are only very few preexisting core clot ship parts to fill them with. 

as to the first questen – a counter question 'how many different kinds of ship beam weapons are necessary' comes to mind, or how many different kinds of planetary environment, or how many native species .. i will not speak those out loud though and keep them to myself.. – and a counter question would not really be an answer anyway, so honestly, i do not know a good answer to that.